src/main/java/com/salesforce/mirus/Mirus.java

```diff
@@ -54,9 +56,12 @@
  */
 public class Mirus {
 
-  private static final Logger logger = LoggerFactory.getLogger(Mirus.class);
+  private static final Logger log = LoggerFactory.getLogger(Mirus.class);
 
-  public static void main(String[] argv) throws Exception {
+  private final Time time = Time.SYSTEM;
```

Comment:
We can remove the `Time` local var in `startConnect()` and use this member var.

Reply:
Good catch @dalassi1  -  to avoid things like that in future I've synchronised even closer with `ConnectDistributed.startConnect` and added links in the JavaDoc. As discussed, ideally we would be using `ConnectDistributed` directly, and that may be possible in the near future.